style.css

```diff
@@ -244,17 +244,44 @@ header {
   z-index: 1;
 }
 
+nav {
+  margin-right: 24px;
+}
+
+.nav {
+  height: 100vh;
+  width: 100vw;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 28px;
+  background-color: rgba(96, 112, 255, 0.9);
+  color: #fff;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
```

Comment:
- [ ] Please consider the design template for styling the menu on the mobile screen!
It's so important to practice following the exact same given design by coding 👍
_Kindly take a look at the two images below and do the corresponding changes like the blurriness behind the nav background and the spacings (margins and padding)._
### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/123716367-b1786800-d87a-11eb-8e8f-c7a89427d41e.png)
### **Original design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/123716372-b9d0a300-d87a-11eb-9916-f0b0c7f9a161.png)